ts/features/pn/analytics/index.ts

```diff
@@ -12,6 +12,13 @@ import {
 import { ThirdPartyAttachment } from "../../../../definitions/backend/ThirdPartyAttachment";
 import { PaymentStatistics } from "../../messages/store/reducers/payments";
 
+export type PNServiceStatus = "active" | "not_active" | "unknown";
+
+export const booleanOrUndefinedToPNServiceStatus = (
+  active: boolean | undefined
+): PNServiceStatus =>
+  active != null ? (active ? "active" : "not_active") : "unknown";
```

Comment:
why check for `null` instead of `undefined` here?

Reply:
I prefer the `!= null` because it covers all possible `undefined`/`nullish` cases. 

Typescript says: "it's always undefined, there's no way it could be `null`" but at runtime such constraints do not hold.

So, in that case, `active !== undefined` would result in `not_active`, with reports being misleading (instead, an unusual amount of `unknown` would trigger some perplexities about them)